### PR TITLE
Index changes and rebases

### PR DESCRIPTION
**Pull request by sudheesh001**
live demo : http://daminisatya.github.io/2016.fossasia.org/

### CODE REVIEW

**Comment by sudheesh001** [comment]
Looks like @mariobehling is adding the images to the repository, this script takes the names from the API url and puts the speakers list up

**Comment by sudheesh001** [comment]
@daminisatya please follow up with subsequent patches and rebases.

**Comment by joerx** [comment]
See my comment on #24

File: index.html [hunk 1]
```diff
@@ -1768,6 +1119,69 @@ <h1 class="text-white">FOSSASIA is the annual OpenTechSummit featuring prominent
         <script src="js/tweetsfetcher.js"></script>
         <script src="js/placeholders.min.js"></script>
         <script src="js/scripts.js"></script>
+        <script>
+        $.ajax({
+	        type : 'GET',
+	        dataType : 'json',
+	        url: 'https://raw.githubusercontent.com/fossasia/open-event-scraper/master/out/speakers.json',
```
Comment (sudheesh001) [inline index.html:1126]: @joerx This is being used as mentioned in #24

**Comment by sudheesh001** [comment]
Good to close because the list is now handcurated and static

**Pull request closed without merging**